extra/nrnivmodl-core.in

```diff
@@ -12,6 +12,16 @@ MAKE_OPTIONS="MECH_NAME NMODL_BINARY DESTDIR INCFLAGS LINKFLAGS MODS_PATH VERBOS
 PARALLELISM=4
 SAVE_FILE="@CMAKE_HOST_SYSTEM_PROCESSOR@/nrnivmodl_options.txt"
 
+# To support build + install later we save/restore options
```

Comment:
is that not a change that is already on master?

Reply:
Yes, but i needed it for testing with spack so I cherrypicked it from the branch before it was merged